lib/plugins/package/lib/packageService.js

```diff
@@ -22,11 +22,11 @@ module.exports = {
   },
 
   getServiceArtifactName() {
-    return `${this.serverless.service.service}.zip`;
+    return `${this.serverless.service.service}.zip`; // NAME_HERE
   },
 
   getFunctionArtifactName(functionObject) {
-    return `${functionObject.name}.zip`;
+    return `${functionObject.name}.zip`; // NAME_HERE
```

Comment:
The // NAME_HERE comments indicate a name, possible name, or possible naming convention and should be removed prior to merge.